packages/squiggle-lang/src/fr/tag.ts

```diff
@@ -46,7 +46,10 @@ function _ensureTypeUsingLambda<T1>(
 }
 
 // This constructs definitions where the second argument is either a type T or a function that takes in the first argument and returns a type T.
-function withInputOrFnInput<T>(inputType: FRType<any>, outputType: FRType<T>) {
+function decoratorWithInputOrFnInput<T>(
```

Comment:
Renamed for clarity. We can rename it back and pass `isDecorator` value explicitly if needed, but I think this function is mostly for `showAs` sake which is always a decorator.